packages/build/src/core/severity.js

```diff
@@ -1,20 +1,20 @@
 'use strict'
 
-// Used for example as exit codes.
+// Used to extract exit codes and respective status strings
 // 1|2|3 indicate whether this was a user|plugin|system error.
 const getSeverity = function (severity = FALLBACK_SEVERITY) {
-  const severityCode = severity in SEVERITY_CODES ? SEVERITY_CODES[severity] : SEVERITY_CODES[FALLBACK_SEVERITY]
+  const severityEntry = severity in SEVERITY_MAP ? SEVERITY_MAP[severity] : SEVERITY_MAP[FALLBACK_SEVERITY]
   const success = severity === SUCCESS_SEVERITY
-  return { success, severityCode }
+  return { success, ...severityEntry }
 }
 
 /* eslint-disable no-magic-numbers */
-const SEVERITY_CODES = {
-  success: 0,
-  none: 1,
-  info: 2,
-  warning: 3,
-  error: 4,
+const SEVERITY_MAP = {
+  success: { severityCode: 0, status: 'success' },
```

Comment:
Should we add a comment that `status` is meant for telemetry?
This would help distinguishing it from the `SEVERITY_MAP` keys (`warning` vs `plugin-error` for example).

Reply:
Addressed in https://github.com/netlify/build/pull/2349/commits/cbef07a8f78fa796aea32bb6ef391783f06f2369, let me know what you think 👍

Reply:
Great!